feature/footer

dmistori84 opened the pull request:
Добавляет футер, но свг не отображается. При добавлении свг с телефоном, пропадает последний див, поэтому он закоментирован

dmistori84 committed:
add footer

AnnaYegorova commented on src/partials/footer.html:
```diff
@@ -1 +1,26 @@
+                тут будет лого
+            </p>
+            
+            <a class="footer__tel" href="+38 (000) 000 00 00">
```
Дописать href="tel:+38   "

AnnaYegorova commented on src/partials/footer.html:
```diff
@@ -1 +1,26 @@
+            
+                +38 (000) 000 00 00
+            </a>
+            <svg class="icon-insta" width="40px" height="40px">
```
Svg оберни в ссылку<a>

AnnaYegorova commented on src/partials/footer.html:
```diff
@@ -1 +1,26 @@
+                +38 (000) 000 00 00
+            </a>
+            <svg class="icon-insta" width="40px" height="40px">
+                <use href="../images/icons/instagram.svg"></use>
```
Путь к картинке в html прописываем через одну точку.

AnnaYegorova commented on src/partials/footer.html:
```diff
@@ -1 +1,26 @@
+        </div>
+        <div class="footer__copy">
+            <p>
+                @ All rights reserved, 2019
```
Спецсимвол заменяешь на &copy;

dmistori84 committed:
Заканчивает верстку и добавляет стили до 479px

СВГ взяты в ссылки, добавлен спецсимвол ©, назначаны стили до 479px

dmistori84 committed:
Добавляет стили на 480 и начало стилей на 768

Закончил стили под мобилку, начал накидывать стили для планшета. Для полноценного выравнивания нужен "лого". Он у меня пока как текст

Vladosik12333 reviewed (commented):
Не внесены правки описанные Анней выше.

dmistori84 committed:
Заканчивает стили 768. Работа над стилями 1200

dmistori84 committed:
Работа на стилями планшета

Выравнивание контактов

dmistori84 committed:
Вставляет свг в ссылку

dmistori84 committed:
Ничегонезнаячащий камит

dmistori84 committed:
Комменті(свг)

dmistori84 committed:
del me

AnnaYegorova commented:
1

AnnaYegorova commented:
1111
